### PR TITLE
Updating Team Member Page

### PR DESCRIPTION
**Pull request by TaraTodd123**
#### Checklist

* [X] The contents of this PR are mine to submit. No third party code or other material is being committed.
* [X] New third party dependencies (if any) are linked via `package.json`. Third party dependencies are licenced under one of the following: Apache, MIT, BSD, Mozilla Public License, Eclipse Public License, or Oracle Binary Code License.
* [X] The code in this PR does not contain export-restricted encryption algorithms.

#### Ticket / Issue

https://portal.digitalsafe.net/browse/EL-3524

#### Description of Proposed Changes

- Created a team member card for myself
- Removed hiring card

#### Documentation CI URL

https://pages.github.houston.softwaregrp.net/sepg-docs-qa/UXAspects_CI_UXAspects_EL-3524-new-team-member

### CODE REVIEW

**Comment by TaraTodd123** [comment]
Removed Alastair and Pearse.

**Comment by TaraTodd123** [comment]
Removed start from navigation bar and added myself to the package.json files

File: package.json [hunk 1]
```diff
@@ -5,10 +5,8 @@
   "main": "docs/app/app.module.ts",
   "contributors": [
     "Ashley Hunter",
-    "Alastair McKee",
-    "Pearse McMurray",
     "Donna Marijne Worby",
-    "Chris Brennan"
+    "Tara Todd"
```
Comment (donna-marijne) [inline package.json]: Ah, could you leave the existing names in place? It's more of a list of people who have contributed significantly to the project, regardless of current status.

Comment (TaraTodd123) [reply package.json]: Re-added

**Pull request merged**